commit 67f6b7e018ad5be83fe9ef881ac627bd6de485d5
Author: Andres Salomon <dilinger@queued.net>
Date:   Fri Oct 21 23:34:00 2022 -0400

    Re-fix TFLite build error on linux when using the system zlib
    
    In commit ae0f9adb7e14c0d19ca695ef6ad40b321a8cb64c, I fixed some build
    errors related to minizip patch inclusion in TFLite. However, after that
    when TFLite Support was rolled to HEAD, a small part of that patch got
    dropped. The result is the following build error with 107.0.5304.62:
    
    ../../third_party/tflite_support/src/tensorflow_lite_support/metadata/cc/utils/zip_readonly_mem_file.cc:22:10: fatal error: 'contrib/minizip/ioapi.h' file not found
             ^~~~~~~~~~~~~~~~~~~~~~~~~
    1 error generated.
    
    This commit re-adds the lost fix.

diff --git a/third_party/tflite_support/patches/0007-Fix-TFLite-build-errors-on-linux-when-using-the-syst.patch b/third_party/tflite_support/patches/0007-Fix-TFLite-build-errors-on-linux-when-using-the-syst.patch
index c8499d374f1ff..80c651f3da909 100644
--- a/third_party/tflite_support/patches/0007-Fix-TFLite-build-errors-on-linux-when-using-the-syst.patch
+++ b/third_party/tflite_support/patches/0007-Fix-TFLite-build-errors-on-linux-when-using-the-syst.patch
@@ -63,6 +63,19 @@ index 13927a7afa698..1b7bed5987fbc 100644
  
  #include "absl/strings/string_view.h"  // from @com_google_absl
 -#include "contrib/minizip/ioapi.h"
++#include "third_party/zlib/contrib/minizip/ioapi.h"
+ 
+ namespace tflite {
+ namespace metadata {
+diff --git a/third_party/tflite_support/src/tensorflow_lite_support/metadata/cc/utils/zip_readonly_mem_file.cc b/third_party/tflite_support/src/tensorflow_lite_support/metadata/cc/utils/zip_readonly_mem_file.cc
+index 392b6b411fe03..525ae4a2b45bd 100644
+--- a/third_party/tflite_support/src/tensorflow_lite_support/metadata/cc/utils/zip_readonly_mem_file.cc
++++ b/third_party/tflite_support/src/tensorflow_lite_support/metadata/cc/utils/zip_readonly_mem_file.cc
+@@ -19,7 +19,7 @@ limitations under the License.
+ #include <cstdio>
+ 
+ #include "absl/strings/string_view.h"  // from @com_google_absl
+-#include "contrib/minizip/ioapi.h"
 +#include "third_party/zlib/contrib/minizip/ioapi.h"
  
  namespace tflite {
diff --git a/third_party/tflite_support/src/tensorflow_lite_support/metadata/cc/utils/zip_readonly_mem_file.cc b/third_party/tflite_support/src/tensorflow_lite_support/metadata/cc/utils/zip_readonly_mem_file.cc
index 392b6b411fe03..525ae4a2b45bd 100644
--- a/third_party/tflite_support/src/tensorflow_lite_support/metadata/cc/utils/zip_readonly_mem_file.cc
+++ b/third_party/tflite_support/src/tensorflow_lite_support/metadata/cc/utils/zip_readonly_mem_file.cc
@@ -19,7 +19,7 @@ limitations under the License.
 #include <cstdio>
 
 #include "absl/strings/string_view.h"  // from @com_google_absl
-#include "contrib/minizip/ioapi.h"
+#include "third_party/zlib/contrib/minizip/ioapi.h"
 
 namespace tflite {
 namespace metadata {
